interface/src/app/gene/network/network.tpl.html

```diff
@@ -0,0 +1,15 @@
+<h1>ADAGE: Gene-Gene Network</h1>
+
+<div ng-hide="ctrl.statusMessage">
```

Comment:
I put more thoughts on this **ng-if** and **ng-hide** question and added a comment to Issue #25 (which I closed yesterday). @rzelayafavila, please let me know whether it makes sense to you.